projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -52,6 +52,17 @@ public function register_rest_routes() {
 			)
 		);
 
+		// WordAds DSP API Posts routes
+		register_rest_route(
+			static::$namespace,
+			sprintf( '/sites/%d/wordads/dsp/api/v1/wpcom/sites/%d/blaze/posts(\?.*)?', $site_id, $site_id ),
```

Comment:
Maybe we can use a numbered placeholder here to avoid having to repeat `$site_id` twice?

```suggestion
			sprintf( '/sites/%1$d/wordads/dsp/api/v1/wpcom/sites/%1$d/blaze/posts(\?.*)?', $site_id ),
```

Reply:
I will try that approach, it sounds better that way. 👍